openpype/hosts/fusion/api/lib.py

```diff
@@ -302,10 +302,25 @@ def get_frame_path(path):
     return filename, padding, ext
 
 
-def get_current_comp():
-    """Hack to get current comp in this session"""
+def get_fusion():
+    """Get current Fusion instance"""
     fusion = getattr(sys.modules["__main__"], "fusion", None)
-    return fusion.CurrentComp if fusion else None
+    return fusion
+
+
+def get_current_comp():
+    """Get current comp in this session"""
+    fusion = get_fusion()
+    if fusion:
+        comp = fusion.CurrentComp
+        return comp
+
+
+def get_comp_filename():
+    """Get comp's Filename"""
+    comp = get_current_comp()
+    if comp:
```

Comment:
```suggestion
    if comp is not NOne:
```

Reply:
I guess you meant to write `if comp is not None:` and not `NOne`